LearningMissionSimulation/PlaygroundMher.cs

```diff
@@ -1,67 +1,118 @@
 {
     public class PlaygroundMher
     {
-        int accountCount;
-        List<string> passwoerdNameList;
-        List<string> userNameList;
-        List<string> emailNameList;
-        List<string> phoneNumberList;
-        List<Enum> roleEnumList;
-        List<Enum> statusEnumList;
-        List<DateTime> createDateList;
-        List<DateTime> updateDateList;
-        public PlaygroundMher(int count)
+        int _accountCount;
+        public PlaygroundMher(int accountCount)
         {
-            this.accountCount = count;
+            this._accountCount = accountCount;
         }
 
-
-        public List<string> PasswoerdNameList { private get => passwoerdNameList; set => passwoerdNameList = value; }
-        public List<string> UserNameList { get => userNameList; set => userNameList = value; }
-        public List<string> EmailNameList { get => emailNameList; set => emailNameList = value; }
-        public List<string> PhoneNumberList { get => phoneNumberList; set => phoneNumberList = value; }
-        public List<Enum> RoleEnumList { get => roleEnumList; set => roleEnumList = value; }
-        public List<Enum> StatusEnumList { get => statusEnumList; set => statusEnumList = value; }
-        public List<DateTime> CreateDateList { get => createDateList; set => createDateList = value; }
-        public List<DateTime> UpdateDateList { get => updateDateList; set => updateDateList = value; }
+        public int count = 0;
+        public int AccountCount { get => _accountCount; set => _accountCount = value; }
 
         Account account = ObjectGenerator.GenerateAccount();
+
+        List<Account> accountList = new List<Account>();
+
+        Queue<DateTime> creatDateTimeQueue = new Queue<DateTime>();
+        
         // Simulate account creation  process
-        public void SimulateAccountCreationProcess(int accountCount)
+        public void CreateAccounts(int accountCount)
         {
             account.Report();
+            Console.WriteLine("\n");
             int i = 0;
             while (i < accountCount)
             {
-                passwoerdNameList.Add(account.Password);
-                userNameList.Add(account.Username);
-                emailNameList.Add(account.Email);
-                phoneNumberList.Add(account.Phone);
-                roleEnumList.Add(account.Role);
-                statusEnumList.Add(account.Status);
-                createDateList.Add(account.CreateDate);
-                updateDateList.Add(account.UpdateDate);
+                accountList.Add(ObjectGenerator.GenerateAccount());
+                i++;
+            }
+            i = 0;
+            foreach (var itemOfAccountList in accountList)
+            {
+                //if (itemOfAccountList.Status == Status.Pending)
+                //{
+                //    creatDateTimeQueue.Enqueue(itemOfAccountList.CreateDate);
+                //}
+                if (!(itemOfAccountList.Status == Status.Active))
```

Comment:
This is not sufficient. Accounts can be active, pending, suspended and closed. I want you to activate accounts that are in pending state.